css/templates/e-commerce/header.css

```diff
@@ -12,6 +12,22 @@
   z-index: var(--z1);
 }
 
+.ampstart-navbar-trigger {
+  cursor: pointer;
+  height: 24px;
```

Comment:
This is hardcoded here since it's the height of the SVG and it allows vertical aligning quite easily.